AEPIdentity/Tests/IdentityTests.swift

```diff
@@ -486,4 +486,49 @@ class IdentityTests: XCTestCase {
         // expect true since identity has booted
         XCTAssertTrue(identity.readyForEvent(firstEvent))
     }
+
+    // analytics shared state not available and hub shared state pending
```

Comment:
Can you add tests for getUrlVariables events.
Also, add tests for AppendToUrl and getUrlVariables where Analytics is not registered, there is no shared state and `readyForEvent` returns true.